internal/lefthook/runner/runner.go

```diff
@@ -14,6 +14,7 @@ import (
 	"sync/atomic"
 
 	"github.com/spf13/afero"
+	"golang.org/x/exp/slices"
```

Comment:
go 1.21 is hot off the press and has in-built support for `slices.Contains`, but I didn't want to bump it without checking - https://go.dev/blog/go1.21

Reply:
@mrexox interpreting your thumbs up to go ahead: I created #536.

do you have any suggestions for test structure for this PR?

Reply:
Hey! Thank you for bringing this feature. I can suggest you to write an integrity test (you can check the existing examples). But it's fine even without tests, I can handle this myself :)